rustybeer-cli/src/commands/alcohol_volume_weight.rs

```diff
@@ -0,0 +1,98 @@
+use clap::{value_t, App, Arg, ArgMatches};
+pub use rustybeer::calculators::alcohol_volume_weight::{
+    calculate_abv_abw, calculate_abv_abw_density, calculate_abw_abv, calculate_abw_abv_density,
+    calculate_alc_vol, calculate_alc_weight,
+};
+
+pub fn add_subcommand<'a, 'b>() -> App<'a, 'b> {
+    App::new("abv_abw")
+        .version("0.1")
+        .author("Joseph Russell (josephrussell123@gmail.com)")
+        .about("Calculates Alcohol by Weight (ABW) from Alcohol By Volue (ABV) and vice versa")
+        .arg(
+            Arg::with_name("percent")
+                .short("p")
+                .long("percent")
+                .value_name("PERCENT")
+                .help("'From' alcohol percentage")
+                .required(true)
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("total_volume")
+                .short("v")
+                .long("total_volume")
+                .value_name("TOTAL_VOLUME")
+                .help("Total beer volume in mL")
+                .required(false)
+                .takes_value(true),
+        )
+        .arg(
+            Arg::with_name("total_density")
+                .short("d")
+                .long("total_density")
+                .value_name("TOTAL_DENSITY")
+                .help("Total density of beer in kg/m³")
```

Comment:
I think there is density unit also in measurements project. I started working on the from_str functionalities there but didn't complete all of them so not sure if I did this. Other option is to create yet another builder for density

Reply:
There seems to only be mass, volume and temperature. I'll see if I can add density to the conversion file in a different ticket.